### PR TITLE
SDI-355 Incentive migration details and cancel

### PR DESCRIPTION
**Pull request by andymarke**
* Details page for monitoring a cancellation
* Cancel endpoint to cancel a running migration

### CODE REVIEW

File: integration_tests/e2e/visitsMigrationDetails.cy.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ context('Visit Migration Details', () => {
         failed: 100,
         stillToBeProcessed: 23100,
       })
-      cy.task('stubGetMigrationDetailsStarted', migrationId)
```
Comment (andymarke) [inline integration_tests/e2e/visitsMigrationDetails.cy.ts:18]: rename, this is specific for visits

File: server/services/nomisMigrationService.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export interface HistoricMigrations {
   migrations: Array<MigrationHistory>
 }
 
-export interface VisitsMigrationDetails {
+export interface HistoricMigrationDetails {
```
Comment (andymarke) [inline server/services/nomisMigrationService.ts:24]: rename, not specific to visits

File: server/views/pages/incentives/startIncentivesMigration.njk [hunk 3]
```diff
@@ -47,7 +47,7 @@
                             name: "fromDate",
                             classes: "govuk-input--width-20",
                             hint: {
-                                text: "Example 2020-03-23. This optional field indicates the used the IEP date not the created date"
```
Comment (andymarke) [inline server/views/pages/incentives/startIncentivesMigration.njk:50]: better wording

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**